Sources/PayPalDataCollector/PayPalDataCollector.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PPRiskMagnes
+
+public class PayPalDataCollector {
+
+    private let magnesSDK: MagnesSDKProtocol
+    private let deviceInspector: DeviceInspectorProtocol
+
+    init(magnesSDK: MagnesSDKProtocol = MagnesSDK.shared(), deviceInspector: DeviceInspectorProtocol = DeviceInspector()) {
+        self.magnesSDK = magnesSDK
+        self.deviceInspector = deviceInspector
+    }
+
+    public func collectDeviceData(additionalData: [String: String] = [:]) -> String {
```

Comment:
I think we could also do something like this in `MagnesSetupParams`:
```
var magnesEnvironment: MagnesSDK.Environment {
    switch CoreConfig.environment {
    case .sandbox:
        return "SANDBOX"
    case .production:
        return "LIVE"
    }
}
```
We could then either have the merchant pass the env in and call that var below or we could just do that for the merchant and pass `CoreConfig.environment.magnesEnvironment` into the params env variable. I think the second would be a little cleaner and the merchant wouldn't have to do anything really, we just grab that from the config for them. (Would also be worth testing this since this is just pseudo code, but I think something like that should work)

Reply:
Ah, actually I think we would need to pass the environment into the init for this class like:
```
public init(config: CoreConfig) {
    self.config = config
    self.magnesSDK = MagnesSDK.shared()
    self.deviceInspector = DeviceInspector()
}
```
Then either do the computed var in the `collectDeviceData` or create something like a `getMagnesEnvironment` that could pass in the env from the config and return the magnes env.

Reply:
Yeah that's a good callout. Would we expect the merchant to instantiate `PayPalDataCollector`? Or is it something that could be done behind the scenes. Either way changing the constructor makes sense I can update it.